pkgs/top-level/all-packages.nix

```diff
@@ -6799,37 +6799,13 @@ with pkgs;
 
   jikes = callPackage ../development/compilers/jikes { };
 
-  julia_04 = callPackage ../development/compilers/julia {
-    gmp = gmp6;
-    openblas = openblasCompat;
-    inherit (darwin.apple_sdk.frameworks) CoreServices ApplicationServices;
-    llvm = llvm_37;
-  };
-
-  julia_05 = callPackage ../development/compilers/julia/0.5.nix {
-    gmp = gmp6;
-    libgit2 = libgit2_0_25;
-    openblas = openblasCompat;
-    inherit (darwin.apple_sdk.frameworks) CoreServices ApplicationServices;
-    llvm = llvm_38;
-  };
-
-  julia_06 = callPackage ../development/compilers/julia/0.6.nix {
+  julia = callPackage ../development/compilers/julia {
```

Comment:
Is there a reason the name `julia_06` was removed?  I have witnessed this oscillate over time and it is frustrating that these names come and go, especially since each minor version of julia before 1.0 is not backwards compatible with others.  At the moment, as of c745ed1f7c5398848c0361e75be3c2d82aaab95b, there is `julia`, `julia_07`, and `julia_10`, but no way of specifying julia 0.6 explicitly.

Reply:
As an aside, I am a pretty serious user (and also a developer of) julia, but only a casual user/developer of nixpkgs.  If having my perspective on julia-related pull requests here in the future would be helpful, feel free to add me to the maintainers list.  (I'm guessing this is the most reliable way to be CCed on any related changes, which would allow me to make comments before things are merged.)

Reply:
I agree on both counts.

Reply:
Julia 0.6 was not removed only julia_04 and julia_05.

Reply:
However I should have kept `julia_06` as an alias.

Reply:
This is already fixed: 0e4197b34bf5dd877817165fedcb9401f019d656

Reply:
I backported this as well:

[detached HEAD 4a62ef08009] julia: readd julia_06 name, make julia an alias that selects a version, as we usually do for incompatible versions
 Author: Michael Raskin <7c6f434c@mail.ru>
 Date: Fri Sep 7 11:36:20 2018 +0300
 1 file changed, 3 insertions(+), 1 deletion(-)

Reply:
I would appreciate if we have active julia maintainer. Then I would feel more comfortable to have multiple versions of julia in nixpkgs. At the time of doing this pull request I had the feeling that no take care of removing old versions, that's why I wanted to shift to having a single version only.